Ex 03

datapawn opened the pull request:
Finalizes ex-03 without revisions.
Finished verifyphone()
Built and finished verifyemail():  note - verifyemail() does not yet check for more than one @ symbol.  That will need to be fixed in future revisions.
Briskette gave me permission to say it was 'done' though.

Commit:
add main, remain, and skratchy modules

Commit:
finished verifyemail() and finished module, ready for next Revision

JayLoomis commented on Exercises/ex-03/base/remain.py:
```diff
@@ -0,0 +1,114 @@
+
+def verifyemail(e):
+    #slice off the last four letters
+    dotcom = e[-4::1]
```
What if their top level domain isn't a three letter one?

Can you think of a less brittle way to get it?

datapawn replied:
Should i change this to accept TLDs other than the standard list provided?  If there is a specific list I can add them, if there is not, i can change the process by which i slice the end of the data off.

JayLoomis commented on Exercises/ex-03/base/remain.py:
```diff
@@ -0,0 +1,114 @@
+    tld_list = (".com",".org",".net",".int",".edu",".gov",".mil")
+
+    #check for @ and .something
+    if "@" not in e:
```
It would be just as easy to count the number of @s and check for 1.

datapawn replied:
So i did that. get off my back.

JayLoomis reviewed (changes requested):
A couple of things to look at.

datapawn committed:
changed verifyemail() #check for @ and .something

Changed the If statement so it verifies there are no more or less than 1 @ symbols.

datapawn reviewed (commented):
Addressed Review Comments Take A Look It's In a Book.

JayLoomis commented:
There are things to do before this is done done, but it's fine for now.

LGTM
